hedera-node/hedera-smart-contract-service-impl/src/main/java/module-info.java

```diff
@@ -16,6 +16,7 @@
     requires com.hedera.node.app.service.token;
     requires com.swirlds.jasperdb;
     requires static com.github.spotbugs.annotations;
+    requires static io.perfmark;
```

Comment:
This was added because the newer `io.grpc` package needs it available at runtime. Is it the correct place, I'm not sure. It was added here to resolve the failures observed in the unit tests from this module in addition to the runtime failures observed in "production" executions.

Reply:
"requires static" does **not** make it available at runtime. If this solved it, something else must have been going on.

If something is missing at runtime you would need `runtimeOnly(gav("..."))` in the "build.gradle.kts". But it should already be there at runtime as  `io.grpcs` pulls it in.

Or maybe it is only a module visibility thing? But then `requires static` although has no effect (at least that's my understanding) ....

Do you still know how to reproduce this and/or how the failure message is?

Reply:
Removing the line resulted in UTs and the actual app execution throwing a module loader exception.

Reply:
Just pushed a build with the line removed.

Reply:
Fixed